components/ui/pathogen-map/pathogen-map.tsx

```diff
@@ -42,9 +46,28 @@ export function PathogenMap<
   generatePopupContent,
   layers,
 }: PathogenMapProps<TPathogenDataPointProperties>) {
-  const [popUpInfo, setPopUpInfo] = useState<
+  const [popUpInfo, _setPopUpInfo] = useState<
```

Comment:
why the underscore?

Reply:
Ahhhh it is denoting that it's private but I want to give it the same name. People do this sometimes to tell people that I want to call `setPopUpInfo` everywhere instead of `_setPopUpInfo`, I think I saw this on some Kotlin tutorial or something lol